tools/gulp/tasks/development.ts

```diff
@@ -28,3 +37,10 @@ task(':serve:devapp', serverTask(outDir, true));
 task('serve:devapp', ['build:devapp'], sequenceTask(
   [':serve:devapp', 'material:watch', ':watch:devapp']
 ));
+
+/** Task that copies all vendors into the demo-app package. Allows hosting the app on firebase. */
+task('build-deploy:devapp', ['build:devapp'], () => {
```

Comment:
Not sure about the name, I feel like it should be `build:[something]`, `build-deploy` makes it sound like it's both building and deploying